package.json

```diff
@@ -10,7 +10,7 @@
     "eyes": "0.1.8",
     "fh-db": "1.4.4",
     "fh-mbaas-client": "0.16.5",
-    "fh-mbaas-express": "5.6.10",
+    "fh-mbaas-express": "^5.9.0",
```

Comment:
better use `~` instead of  `^` to lock down to patch version

Reply:
I agree with this, making the change